docs/gettingstarted.rst

```diff
@@ -0,0 +1,110 @@
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| default?    | bool                                       | the first required option for the user to complete--only one option can be default                 |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| required?   | bool                                       | if the parameter is required or optional--default ``false``                                        |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| choices?    | array of `ApplicationCommandOptionChoice`_ | choices for ``string`` and ``int`` types for the user to pick from                                 |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| options?    | array of `ApplicationCommandOption`_       | if the option is a subcommand or subcommand group type, this nested options will be the parameters |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+
+This table shows us the way that Discord handles the structure of options for
+slash commands through their Bot API. For visualization purposes, we'll quickly
+make a JSON example (although we won't be using it) in order to explain how this
+works:
+
+.. code-block:: python
+
+  {
+    "name": "argOne",
+    "description": "description of first argument",
+    "type": 3, # STRING type,
+    "required": True
+  }
+  
+With this very basic understanding in mind, now we are able to begin programming
+a simple Python script that will allow us to utilize this ability through one of
+the many subclasses offered in *discord-py-slash-command*.
+
+.. code-block:: python
+
+  import discord
+  from discord_slash import SlashCommand
+  from discord_slash.utils import manage_commands # Allows us to manage the command settings.
+
+  client = discord.Client(intents=discord.Intents.all())
+  slash = SlashCommand(client, auto_register=True)
+
+  guild_ids = [789032594456576001]
+
+  @client.event
+  async def on_ready():
+      print("Ready!")
+
+  @slash.slash(
+    name="test",
+    description="this returns the bot latency",
+    options=[manage_commands.create_option(
+      name = "argOne",
+      description = "description of first argument",
+      option_type = 3, # STRING type,
+      required = True
+    )],
+    guild_ids=guild_ids
+  )
+  async def _test(ctx, argOne: str):
```

Comment:
I would suggest using `arg_one` instead of `argOne` as it is pythons naming convention.